src/packet/BinaryWriter.js

```diff
@@ -1,137 +1,62 @@
-﻿'use strict';
+'use strict';
 /*
  * Simple BinaryWriter is a minimal tool to write binary stream with unpredictable size.
  * Useful for binary serialization.
  *
- * Copyright (c) 2016 Barbosik
- * 
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- * 
- *     http://www.apache.org/licenses/LICENSE-2.0
- * 
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- * 
- * 
+ * Copyright (c) 2016 Barbosik https://github.com/Barbosik
+ * License: Apache License, Version 2.0
  */
 
-function BinaryWriter(size) {
-    if (!size || size <= 0) {
-        size = Buffer.poolSize / 2;
-    }
-    this._buffer = new Buffer(size);
+function BinaryWriter() {
+    this._buffer = parseInt(process.version[1]) < 6 ? new Buffer(1e5) : Buffer.allocUnsafe(1e5);

```

Comment:
> Your c projects are useless. Other people have probably also made them too. And no one uses VBA.

Wow, making peoples projects look bad. That's not really a "company friendly" thing to do now is it? VBA is actually the main starting point for many starter developers. (Just like Python) I started with Visual Basic when I was 10, just messing with buttons. Just because production programs aren't made in it, doesn't mean it is useless. Other people also made Ogar already, so I guess OpenAgar is also useless?

> And I dont care if you know a friend that can do it. You dont. You wasted your life.

No one cares that you like to argue and shit-talk others on GitHub.

> And when did my rant point to OpenAgar? OpenAgar can anyway support it.

Put it on a production server and get 1000 bound connections to it, then I'll believe you.